tests/e2e/scale/test_scale_12_OCS_worker_nodes_and_6000_PVCs.py

```diff
@@ -75,15 +75,16 @@ def test_scale_node_and_capacity(self):
                 scale_worker_count = expected_worker_count - len(
                     existing_ocs_worker_list
                 )
-                assert scale_lib.scale_ocs_node(node_count=scale_worker_count)
+                scale_lib.scale_ocs_node(node_count=scale_worker_count)
 
             # Check existing OSD count and add OSDs if required
             if existing_deviceset_count < expected_deviceset_count:
-                add_deviceset_count = (
+                add_deviceset_count = int(
```

Comment:
can you pls change this variable name to improve readability in line#86

Reply:
@keesturam I have below example part of the function(scale_capacity_with_deviceset) call https://github.com/red-hat-storage/ocs-ci/blob/742dd680cfc7b2acd4fca3103f71e04a64a599ea/ocs_ci/ocs/scale_lib.py#L1562
`    Example: If add_deviceset_count = 2 & existing storagecluster
    has deviceset count as 1, at the end of function deviceset value
    will be existing value + add_deviceset_count i.e. 3`

Reply:
maybe `additional_deviceset`?

Reply:
updated the code.